spec/javascripts/services/listing_service_spec.coffee

```diff
@@ -7,16 +7,16 @@ do ->
     httpBackend = undefined
     fakeListings = getJSONFixture('/listings.json')
     fakeListing = getJSONFixture('/listings/0.json')
-    $cookies = undefined
+    $localStorage = undefined
```

Comment:
@chrisdolendo Good Q, yeah it is super clean, you don't need any getters and setters ("No Getter 'n' Setter Bullshit" as it says on [their github](https://github.com/gsklee/ngStorage)) which makes the 2-way binding super duper easy, and since that's what I used on income calculator I figured it is cleaner to refactor to use the same "locally stored data" solution for favorites too rather than sometimes $localStorage, sometimes $cookies.